Resources/Prototypes/Catalog/VendingMachines/nutri.yml

```diff
@@ -3,3 +3,14 @@
   name: NutriMax
   description: A vending machine containing nutritional substances for plants and botanical tools.
   spriteName: nutri
+  startingInventory:
+    Spade: 3
+    PlantBGoneSpray: 20
+    WeedSpray: 20
+    PestSpray: 20
+    #syringe
+    #cultivator
```

Comment:
should add like "TODO: add missing prototypes" to the files that have stuff commented out so it's easier to find them